guides/doc-Configuring_Load_Balancer/topics/Promoting_SCAP_Content_to_Clients.adoc

```diff
@@ -5,7 +5,7 @@ The following section describes how to promote Security Content Automation Proto
 
 .Prerequisites
 
-* Ensure that you configure the SCAP content. For more information, see link:{BaseURL}administering_red_hat_satellite/chap-red_hat_satellite-administering_red_hat_satellite-security_compliance_management#configuring_scap_content[Configuring SCAP Content] in _Administering {ProjectName}_.
+* Ensure that you configure the SCAP content. For more information, see {AdministeringDocURL}configuring_scap_content[Configuring SCAP Content] in _Administering {ProjectName}_.
```

Comment:
This is because this section does not have an ID, so Asciidoc gives it an ID itself, and it does it inconsistently. Downstream , the ID is `#configuring_scap_content`, and upstream it is `#_configuring_scap_content` with an extra underscore. Fixing this.

Reply:
Note that my fix will break GitHub checks but will work fine after the PR is merged.

Reply:
I pulled your latest changes... I get https://docs.theforeman.org/master/Administering_Red_Hat_Satellite/index-foreman.html#sect-Red_Hat_Satellite-Administering_Red_Hat_Satellite-Security_Compliance_Management_with_OpenSCAP-Configuring_SCAP_Content

Reply:
Still not working ?

Reply:
This is correct, the link you pastd will work after this PR is merged and new doc is published. I gave a new ID to the section, and refer to this ID.

Reply:
Okidoke :)